presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewInformationExtractor.java

```diff
@@ -101,6 +101,7 @@ public MaterializedViewInfo getMaterializedViewInfo()
         private Optional<Relation> baseTable = Optional.empty();
         private Optional<Expression> whereClause = Optional.empty();
         private Optional<Set<GroupingElement>> groupBy = Optional.empty();
+        private Optional<Set<Expression>> expressionsInGroupBy = Optional.empty();
```

Comment:
It seems to me that we can manage everything with expressionsInGroupBy, we do not need groupBy field.